### ISSUE TITLE
Errata for 2.5.5 Target Size (Enhanced) to align with  2.5.8

### ISSUE THREAD

**Issue by alastc** [w3c/wcag#2857]
There is a proposed update in #2856 for Target Size (min) to the "inline" exception:
"The target is in a sentence, or the size of the target is otherwise constrained by the line-height of non-target text;"

I think this is a clearer and more testable version, and would like to update SC 2.5.5 in WCAG 2.1 on that basis.

**Comment by GreggVan** [comment]
Good except can we make it  "line spacing" instead of "line height" to account for vertical languages? 
gregg

**Comment by alastc** [comment]
Hi Gregg,

In the discussion on Friday people generally found "line height" easier to understand than "inter-line spacing". 

We didn't discuss line-spacing, but (putting the exact typographical definitions to one side), the size of a link in a line of text wouldn't be constrained by the spacing around the line. It would be constrained by the size of the line it is within.

In the PR (#2858) for the AA version there is a note underneath: 
"For inline targets the line-height should be interpreted as perpendicular to the flow of text. For example, in a language displayed top to bottom, the line-height would be horizontal."

Does that work for you?

**Comment by GreggVan** [comment]
Yes -  I see you added a note to that effect.   That works for me.
Thanks 
gregg

**Comment by GreggVan** [comment]
Yep
Works great

**Comment by patrickhlauke** [comment]
xref #3086

**Comment by fstrr** [comment]
Is this issue still being worked on? There's a [related branch from December 2022](https://github.com/w3c/wcag/pull/2858) that is up for deletion unless it's needed.

**Comment by patrickhlauke** [comment]
ping @alastc ... as this seems to relate to WCAG 2.1 ... is anything happening here still and can it be closed?

**Comment by patrickhlauke** [comment]
I believe this issue related to https://github.com/w3c/wcag/pull/2856 which has been merged. #2858 was mentioned later in the discussion here, but that is currently marked as a potential future erratum for 2.3. Closing this issue.
